src/bearbyWallet/BearbyAccount.ts

```diff
@@ -164,22 +180,23 @@ export class BearbyAccount implements IAccount {
       );
     }
 
-    let unsafeParameters: Uint8Array;
-    if (parameter instanceof Uint8Array) {
-      unsafeParameters = parameter;
-    } else {
-      unsafeParameters = Uint8Array.from(parameter.serialize());
+    const unsafeParameters =
+      parameter instanceof Uint8Array
+        ? parameter
+        : Uint8Array.from(parameter.serialize());
+    let operationId;
+    try {
+      operationId = await web3.contract.call({
+        maxGas: Number(maxGas),
+        coins: Number(amount),
+        fee: Number(fee),
+        targetAddress: contractAddress,
+        functionName: functionName,
+        unsafeParameters,
+      });
+    } catch (error) {
+      throw operationErrorMapping('callSC', error);
```

Comment:
can youuse or  create a enum for every transaction type . (callSC, sendTransaction etc... ) to avoid string like this